components/breadcrumb/overview.md

```diff
@@ -0,0 +1,119 @@
+
+>caption In this article:
+
+* [Basics](#basics)
+* [Features](#features)
+
+
+## Basics
+
+To use a Telerik Breadcrumb for Blazor:
+
+1. add the `TelerikBreadcrumb` tag
+1. provide a collection of models to its `Data` property (read more in the [Data Binding article]({%slug breadcrumb-data-binding%}))
+1. match the fields in the models with the binding schema for the nodes
+    * In this example, we keep it simple by only providing text for the Breadcrumb items. See the [Navigation article]({%slug breadcrumb-navigation%}) for more details on how to use the Breadcrumb for navigating through items.
+
+>caption Basic Breadcrumb with data binding. The result from the snippet below.
+
+![Breadcrumb for Blazor](images/breadcrumb-basic-example.png)
+
+````CSHTML
+@* This example demonstrates the basic configuration of the Breadcrumb*@
+
+<TelerikBreadcrumb Data="@Items">
+</TelerikBreadcrumb>
+
+@code {
+
+    public List<BreadcrumbItem> Items { get; set; }
+
+    protected override void OnInitialized()
+    {
+        Items = new List<BreadcrumbItem>
+        {
+            new BreadcrumbItem { Text = "Home", Icon = "home" },
+            new BreadcrumbItem { Text = "Products"},
+            new BreadcrumbItem { Text = "Computer peripherals"},
+            new BreadcrumbItem { Text = "Keyboards"},
+            new BreadcrumbItem { Text = "Gaming keyboards"}
+        };
+    }
+
+    public class BreadcrumbItem
+    {
+        public string Text { get; set; }
+        public string Icon { get; set; }
+        public string Url { get; set; }
+    } 
+}
+````
+
+
+>caption Component namespace and reference
+
+````CSHTML
+<TelerikBreadcrumb @ref="theBreadcrumbRef" Data="@Items">
+</TelerikBreadcrumb>
+
+@code{
+    Telerik.Blazor.Components.TelerikBreadcrumb<BreadcrumbItem> theBreadcrumbRef { get; set; }
+
+    public IEnumerable<BreadcrumbItem> Items { get; set; }
+
+    protected override void OnInitialized()
+    {
+        Items = new List<BreadcrumbItem>
+        {
+        new BreadcrumbItem { Text = "Item1"},
+        new BreadcrumbItem { Text = "Item2"},
+        new BreadcrumbItem { Text = "Item3"}
+        };
+    }
+
+    public class BreadcrumbItem
+    {
+        public string Text { get; set; }
+        public string Icon { get; set; }
+        public string Url { get; set; }
+    }
+}
+````
+
+## Features
+
+The Breadcrumb provides the following features:
+
+* `Data` - a collection of flat data for all items in the Breadcrumb. See the [Data Binding]({%slug breadcrumb-data-binding%}) article for details.
+
+* `CollapseMode` - specifies the collapse mode of the Breadcrumb component. Read more in the [Collapse Modes]({%slug breadcrumb-collapse-modes%}) article.
```

Comment:
We should try to avoid explaining a property with the same words.

```suggestion
* `CollapseMode` - specifies how the Breadcrumb items are displayed if they cannot fit on a single line. Read more in the [Collapse Modes]({%slug breadcrumb-collapse-modes%}) article.
```